### PR TITLE
#19: Episode post type

### LINKED ISSUE #19: Configure Episode Post type
See: https://docs.google.com/spreadsheets/d/10b6bA1-RHGuNgPtczYhJJWorjS5_kvnsIBscqeJi_Qw/edit#gid=746853851

### PR DESCRIPTION
**Pull request by brandonhundt**
Closes #19 

- Creates the episodes custom post type and sets up custom fields

## To Review

- [x] Checkout Branch.
- [x] Run `npm run local` (or `npm run refresh` if you want a fresh database.)
- [x] Go to http://the-world-wp.lndo.site/.

> ...then...

- [x] Copy a TW episode from the existing site.
- [x] Go into the WP-Admin, and select Episode from the menu. Create a new episode.
- [x] Copy in the contents of the episode to the best of your ability.
- [x] Confirm all fields we need for the migration are set up. [Reference this spreadsheet](https://docs.google.com/spreadsheets/d/10b6bA1-RHGuNgPtczYhJJWorjS5_kvnsIBscqeJi_Qw/edit#gid=746853851)
- [x] Review episode API endpoint https://the-world-wp.lndo.site/wp-json/wp/v2/episode

### CODE REVIEW

File: wp-content/themes/newspack-theme/acf-json/group_622a0f70c890f.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "key": "group_622a0f70c890f",
+    "title": "Episode Audio",
+    "fields": [
+        {
+            "key": "field_622a102fdd5e8",
+            "label": "Audio",
+            "name": "audio",
+            "type": "clone",
```
Comment (brandonhundt) [inline wp-content/themes/newspack-theme/acf-json/group_622a0f70c890f.json:9]: ACF has an option to reference an already existing field in the db. Did that here for audio and broadcast date.

File: wp-content/themes/newspack-theme/acf-json/group_622a159a3f6be.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+    "key": "group_622a159a3f6be",
+    "title": "Related Stories",
+    "fields": [
+        {
+            "key": "field_622a15ada3493",
+            "label": "Stories in this episode",
```
Comment (brandonhundt) [inline wp-content/themes/newspack-theme/acf-json/group_622a159a3f6be.json:7]: Hopefully, this allows us to create a migration path for anything thats selected already. 

But maybe not necessary???

**Review by rpeterman-gp — CHANGES_REQUESTED** [review]

File: wp-content/themes/newspack-theme/acf-json/group_622a159a3f6be.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                "class": "",
+                "id": ""
+            },
+            "post_type": [
+                "post"
+            ],
+            "taxonomy": "",
+            "filters": [
+                "search",
+                "post_type",
```
[the remark this answers is not in the document]
Comment (brandonhundt) [reply wp-content/themes/newspack-theme/acf-json/group_622a159a3f6be.json]: Fixed!

File: wp-content/themes/newspack-theme/acf-json/group_622a159a3f6be.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
+{
+    "key": "group_622a159a3f6be",
+    "title": "Related Stories",
+    "fields": [
+        {
+            "key": "field_622a15ada3493",
+            "label": "Featured Episodes",
```
[the remark this answers is not in the document]
Comment (brandonhundt) [reply wp-content/themes/newspack-theme/acf-json/group_622a159a3f6be.json]: I changed it to Related Stories, good catch.

**Review by rpeterman-gp — APPROVED** [review]
Nice work! :shipit:

**Pull request merged**